src/opera/api/controllers/manage_controller.py

```diff
@@ -125,9 +121,10 @@ def post_manage_csar(blueprint_token, revision_msg=None):
 
     :rtype: GitRevisionMetadata
     """
-    if not CSAR_db.check_token_exists(blueprint_token):
-        return JustMessage("Blueprint token does not exist, 'manage' route instead"), 404
     file = connexion.request.files['CSAR']
+
+    blueprint_info = SQL_database.get_git_transaction_data(blueprint_token)
```

Comment:
What's the point of this line? blueprint_info appeared to be unused.

Reply:
No need. Refactoring glitch